apps/web/src/app/host-dashboard/page.tsx

```
@@ -0,0 +1,1131 @@
+            <p className="text-sm dark:text-white text-gray-600">Earned</p>
+          </div>
+        </div>
+        
+        <div className="flex space-x-2">
+          <button 
+            onClick={() => {
+              setSelectedProperty(property);
+              setShowCalendarModal(true);
+            }}
+            className="flex-1 bg-blue-600 text-white px-4 py-2 rounded-lg hover:bg-blue-700 transition-colors flex items-center justify-center"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)